ksqldb-execution/src/test/java/io/confluent/ksql/execution/codegen/SqlToJavaVisitorTest.java

```diff
@@ -1149,15 +1148,6 @@ public void shouldThrowOnSimpleCase() {
     );
   }
 
-  @Test
-  public void shouldThrowOnTimeLiteral() {
-    // When:
```

Comment:
Are there any positive tests we can have in this class?

Reply:
Added a test